### DISCUSSION TITLE
Mask 组件在部分安卓机下滚动穿透

### DISCUSSION THREAD

**Discussion by MaxCDon** [ant-design/ant-design-mobile#5615] [Q&A]
代码：https://codesandbox.io/s/jolly-bhabha-6gw6hz?file=/src/App.tsx

复现页面：https://6gw6hz.csb.app/

问题机型：华为 nova8 Pro、华为荣耀9X、华为P30、OPPO R17 Pro、小米12等

https://user-images.githubusercontent.com/16526078/186611902-9b4982db-3e44-4a3f-853a-1c9ad6bd883c.mov

**Comment by mochengdx** [comment]
添加自定义样式 `touchAction: 'none'` @MaxCDon

**Reply by MaxCDon** [reply]
定位到问题是部分安卓机双指滚动导致穿透，`touchAction: 'none'`  添加此样式也无效
